conf/extra-modules-example.yaml

```diff
@@ -201,6 +205,16 @@ modules:
 # value is 50 and the hard cap is 100.
 #chanhistory_maxlines: 50
 
+# CommandAlias Configuration
+# This module requires a dictionary that specifies the aliases and the commands
+# they are replacing. The special value "%p" can be used to specify where the
+# given command parameters for the alias should be.
+# The given example would result in:
+# Input: user!ident@host HELP :I would like some help.
+# Output: user!ident@host PRIVMSG UsefulOper :I would like some help.
+#command_aliases:
+#  HELP: PRIVMSG UsefulOper %p
```

Comment:
To clarify: In this example, you're OK with users having to type "/help :I need help!" complete with the colon? (Or, more commonly, since /help is an alias for local help in almost every client, /quote help :I need help!)